src/pages/chainlink-automation/tutorials/nft-quickstart.mdx

```diff
@@ -0,0 +1,113 @@
+
+This template uses the [Chainlink Automation](https://automation.chain.link) decentralized network to automate the reveal process so you don't have to stand up and maintain an inhouse system for automation.
+
+{/* prettier-ignore */}
+<div class="remix-callout">
+  <a href="https://github.com/smartcontractkit/chainlink-automation-templates/tree/main/batch-nft-reveal">See the code on GitHub</a>
+</div>
+
+<Aside type="caution" title="Disclaimer">
+  This tutorial is an example to help you learn how to integrate Chainlink services and integrate them into your own
+  dApps. This template is provided "AS IS" without warranties of any kind, has not been audited, and may be missing key
+  checks or error handling to make the usage of the product more clear. Do not use the code in this example in a
+  production environment without completing your own audits and application of best practices.
+</Aside>
+
+## Overview
+
+In this tutorial, you will use a [demo user interface](https://automation.chainlink-demo.app) to learn how to use Chainlink Automation to automate batch reveal of NFT mints. The demo includes setting up a Chainlink VRF subscription, funding it, automatically generating a smart contract that will then be used by both VRF (as a consumer) and in Automation (as the target contract). VRF is used for randomizing the metadata for the NFTs and Automation is used to set trigger conditions for when the NFT metadata is revealed via batch size and/or time interval parameters
+
+## Before you begin
+
+Before you start this tutorial, complete the following items:
+
+- Set up a cryptocurrency wallet
+- To deploy this contract on testnets, ensure the deployer account has testnet ERC-677 LINK. Use the [LINK faucet](https://faucets.chain.link/) to retrieve testnet LINK.
+
+## Run steps
+
+1. Go to the [demo app](https://automation.chainlink-demo.app/) and connect to your wallet with the right network. Then click on the **Get Started** button.
+
+   <ClickToZoom
+     src="/images/automation/tutorials/batch-nft1.png"
+     alt="Batch NFT Quickstart Step 1"
+     style="max-width: 70%;"
+   />
+
+1. Enter the basic information regarding your NFT collection. You can hover over the question mark icon to learn more about each field. Notice that you'll need a VRF Subscription ID.
+
+   <ClickToZoom
+     src="/images/automation/tutorials/batch-nft2.png"
+     alt="Batch NFT Quickstart Step 2"
+     style="max-width: 70%;"
+   />
+
+1. At this stage, you will need to create a VRF subscription and fund it with LINK by visiting the [Chainlink VRF Subscription Manager](https://vrf.chain.link/). Follow the steps outlined in the interface to create a subscription. You will receive a confirmation message upon successful creation.
+
+1. Add funds to your subscription. Copy the resulting subscription ID to your clipboard.
```

Comment:
Include a good starting point for how much LINK someone might need to fund their VRF subscription. (Although it is shown in the screenshot, also add it to the text here for more clarity and accessibility.)

Reply:
Resolved